apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerConstants.java

```diff
@@ -225,4 +227,10 @@
    */
   static final String CODING_BBAPI_BENE_HICN_HASH_OLD =
       "http://bluebutton.cms.hhs.gov/identifier#hicnHash";
+
+  /**
+   * Fallback value to use when a record does not have a lastUpdated value. These records where
+   * loaded before the lastUpdated feature was in place.
+   */
+  static final Date FALLBACK_LAST_UPDATED = Date.from(Instant.parse("2020-01-01T00:00:00Z"));
 }
```

Comment:
This is an arbitrary date that is before any actual lastUpdated values. I've chosen the first second of the year, to look arbitrary.